package.json

```diff
@@ -31,6 +31,7 @@
     "truffle": "^4.1.14",
     "typedarray-to-buffer": "^3.1.5",
     "web3": "1.0.0-beta.36",
+    "yaeti": "^1.0.2",
     "zos-lib": "1.4.1"
```

Comment:
Why do we need this `yeati` package?

Reply:
because Travis failed tests without this package :man-shrugging: